smalltalksrc/VMMaker/VMStackPage.class.st

```diff
@@ -34,6 +34,17 @@ VMStackPage class >> alignedByteSizeOf: anObject forClient: aVMClass [
 	^self surrogateClass alignedByteSize
 ]
 
+{ #category : #translation }
+VMStackPage class >> filteredInstVarNames [
+
+	| instVarNames |
+	instVarNames := super filteredInstVarNames.
+
+	^ BytesPerWord = 4
+		  ifTrue: [ ^ instVarNames copyWithout: #padToWord ]
+		  ifFalse: [ ^ instVarNames ]
```

Comment:
```suggestion
	^ BytesPerWord = 4
		  ifTrue: [ instVarNames copyWithout: #padToWord ]
		  ifFalse: [ instVarNames ]
```

Reply:
No need to have returns and returns. conditionals are expressions is Pharo, so they return a value.
Here, either we put a single return outside, or one return in each branch, but not both flavours :)

I personally prefer the first version